packages/astro/src/core/config/schema.ts

```diff
@@ -180,35 +176,6 @@ export const AstroConfigSchema = z.object({
 	legacy: z.object({}).optional().default({}),
 });
 
-interface PostCSSConfigResult {
-	options: Postcss.ProcessOptions;
-	plugins: Postcss.Plugin[];
-}
-
-async function resolvePostcssConfig(inlineOptions: any, root: URL): Promise<PostCSSConfigResult> {
```

Comment:
This function is unused. The `postcss` and `postcss-load-config` deps can be removed along.